### ISSUE TITLE
Brazil GTM - Localize Webmaker to pt-br

### ISSUE THREAD

**Issue by panaggio** [mozilla/webmaker-launch#52]
Goal:
- Webmaker and other related assets fully localized

Team: Localization & QA

**This needs to be done before FISL** (chosen launch date in Brazil), blocks https://github.com/mozilla/webmaker-launch/issues/72 (and several others)

Stuff to be localized are listed here: https://github.com/mozilla/webmaker-launch/wiki/Localization

~~Neither https://github.com/mozilla/webmaker-android nor https://github.com/mozilla/webmaker-desktop are localizable by now (https://github.com/mozilla/webmaker-android/issues/2002, https://github.com/mozilla/webmaker-desktop/issues/66). Until they are, this issue is blocked.~~

https://github.com/mozilla/webmaker-desktop, Play Store strings and new Webmaker landing page  aren't localizable yet, so this part of the issue is blocked.

Blocked tasks:
- [ ] Localize webmaker-desktop
- [ ] QA webmaker-desktop localization
- [ ] Localize Play Store strings
- [ ] QA Play Store strings localization
- [ ] Localize Webmaker landing page
- [ ] QA Webmaker landing page localization

Tasks:
- [x] Localize webmaker-android
- [ ] QA webmaker-android localization
- [x] Localize webmaker-mailroom
- [ ] QA webmaker-mailroom localization

**Comment by secretrobotron** [comment]
Here's a wiki with information about each part of Webmaker that needs localizing: https://github.com/mozilla/webmaker-launch/wiki/Localization

**Comment by panaggio** [comment]
@secretrobotron About webmaker-android and webmaker-mailroom: are them string frozen? I've already localized everything, but I'm not sure it I should watch those files on Transifex for changes or if I can expect this to be really done.

**Comment by secretrobotron** [comment]
@panaggio yes, they're string frozen. We're going to try for a regular release schedule (every two weeks?).

**Comment by panaggio** [comment]
I'll watch those files and ask here for localization when something new show up.

Is the QA "script" frozen too? We still need to check if our current localization is OK on the app.

Also, is there a quick guide for building the app for localization? If not, we need one, so people that are willing to do QA can do it on their computers. It's the same for building the app for development, with an additional step: downloading current strings from Transifex and applying them before building.

**Comment by panaggio** [comment]
And where are the release going to be planned? Here or on https://github.com/mozilla/webmaker-android? We could add an additional step on the release plan to have sign offs from each localization QA team. What do you think?

**Comment by secretrobotron** [comment]
@panaggio sorry for the long pong. Moving foward, I think that's a great idea. Do you think we could set up a localization QA session this week?

Also, we've just uploaded another transifex file to handle the desktop site: https://www.transifex.com/projects/p/webmaker/resource/betawebmakerorg/

cc @panaggio @rodrigowaters @melissacd
